config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/config/loader/EntityLoaderBase.java

```diff
@@ -44,6 +46,10 @@ public Object loadSingle(String name, File entitiesFile) {
         return loadEntitiesFromFile(jsonTools, this.getBeanClass(), entitiesFile).get(name);
```

Comment:
This can be simplified to `return load(entitiesFile).get(name)` now.

Reply:
Done